### PR TITLE
whisper: move whisper to another repo

### PR DESCRIPTION
**Pull request by gballet**
Move Whisper to [its own repo](github.com/ethereum/whisper).

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -8,59 +8,60 @@ require (
...
+	github.com/gballet/go-libpcsclite v0.0.0-20191108122812-4678299bea08
 	github.com/go-ole/go-ole v1.2.1 // indirect
 	github.com/go-sourcemap/sourcemap v2.1.2+incompatible // indirect
 	github.com/go-stack/stack v1.8.0
-	github.com/golang/protobuf v1.3.2-0.20190517061210-b285ee9cfc6c
+	github.com/golang/protobuf v1.3.2
 	github.com/golang/snappy v0.0.2-0.20200707131729-196ae77b8a26
 	github.com/google/go-cmp v0.3.1 // indirect
-	github.com/gorilla/websocket v1.4.1-0.20190629185528-ae1634f6a989
+	github.com/gorilla/websocket v1.4.1
```
Comment (holiman) [inline go.mod:32]: This change is actually going from 1.4.1+ back to 1.4.1, no? 
I think in general, you seem have updated the go modules in 'wildcard mode'? While that may be good (or not), I think deps-updates should be done more deliberately in a separate PR.

File: cmd/geth/main.go [hunk 2]
```diff
@@ -193,13 +193,6 @@ var (
 		utils.RPCGlobalTxFeeCap,
 	}
 
-	whisperFlags = []cli.Flag{
-		utils.WhisperEnabledFlag,
-		utils.WhisperMaxMessageSizeFlag,
-		utils.WhisperMinPOWFlag,
-		utils.WhisperRestrictConnectionBetweenLightClientsFlag,
-	}
```
Comment (holiman) [inline cmd/geth/main.go:201]: It might be better to leave the flags, but put them in 'DeprecatedFlags' and just issue warnings whenever any one of them is used. "The flag xxx has been deprecated; whisper has been moved to a separate repository (link)." Otherwise nodes which uses these flags will just fail to come up again after an update.

**Comment by gballet** [comment]
Replaced by #21487.
